src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/AggregateSymbol.cs

```diff
@@ -109,19 +109,16 @@ public bool FindBaseAgg(AggregateSymbol agg)
             return false;
         }
 
-        public NamespaceOrAggregateSymbol Parent
-        {
-            get { return parent.AsNamespaceOrAggregateSymbol(); }
-        }
+        public NamespaceOrAggregateSymbol Parent => parent as NamespaceOrAggregateSymbol;
 
         public bool isNested()
         {
-            return parent != null && parent.IsAggregateSymbol();
+            return parent is AggregateSymbol;
         }
 
         public AggregateSymbol GetOuterAgg()
```

Comment:
I agree, and have been mostly converting one-liners as I come to them (or as I reduce multi-liners into one-liners). I clearly missed this one though.